package.json

```diff
@@ -18,23 +18,20 @@
     ]
   },
   "scripts": {
-    "clean": "rimraf dist && bolt ws exec -- rimraf dist",
-    "prebuild": "bolt clean && ts-node tools/link-ts.ts && ts-node tools/copy-shared-types.ts && tsc --project ./tsconfig.packages.json --emitDeclarationOnly && ts-node tools/copy-types.ts",
-    "build": "yarn build:quick",
-    "build:quick": "bolt ws exec -- swc src --out-dir dist --quiet --extensions \".ts\" --config-file ../../../.swcrc",
+    "clean": "rimraf dist && bolt ws exec -- rimraf dist tsconfig.tsbuildinfo",
+    "build": "bolt ws exec -- swc src --out-dir dist --quiet --extensions \".ts\" --config-file ../../../.swcrc",
+    "build:full": "bolt ws exec -- tsc -b",
```

Comment:
I changed the set of build commands from `build:quick` and `build` (which just calls the former) to `build` (same as `build:quick` before) and now `build:full` which makes a full typescript build, along with generated types, which I think should be in the distributed packages.